test/cmdlineTests/external_function_pointers_taking_calldata_args/input.json

```
@@ -0,0 +1,13 @@
+{
+    "language": "Solidity",
+    "sources": {
+        "url_not_found.sol": {
+            "urls": [
+                "/projects/contracts.sol",
+                "contract.sol",
+                "/tests/test.sol"
```

Please don't use absolute paths here they could exist on someone's machine.

Use paths relative to the current test dir, since we can assume that there are no stray files in it. For example:

```suggestion
                "external_function_pointers_taking_calldata_args/non-existent-contract-1.sol",
                "external_function_pointers_taking_calldata_args/contract.sol",
                "external_function_pointers_taking_calldata_args/non-existent-contract-2.sol"
```